### PR TITLE
feature/add_movies_query

### PR DESCRIPTION
**Pull request by yaorlov**
https://garage.atlassian.net/secure/RapidBoard.jspa?rapidView=36&projectKey=PEO&selectedIssue=PEO-938

### CODE REVIEW

File: app/concepts/movie/operation/show.rb [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+
+module Movie::Operation
+  class Show < Trailblazer::Operation
+    step Model(Movie, :find_by)
+    step :set_result
+
+    def model(ctx, params:, **)
+      ctx[:model] = Movie.find_by(id: params[:id])
+    end
```
Comment (bl0rch1d) [inline app/concepts/movie/operation/show.rb]: :slightly_smiling_face:

File: app/graphql/resolvers/auth_base.rb [hunk 2]
```diff
@@ -2,7 +2,7 @@
 
 module Resolvers
   class AuthBase < Base
-    def authorized?(**_kwargs)
+    def authorized?(**_args)
```
Comment (bl0rch1d) [inline app/graphql/resolvers/auth_base.rb]: why args?
args - single value arguments
kwargs - key-value arguments

Comment (yaorlov) [reply app/graphql/resolvers/auth_base.rb]: as for me `**args` and `**kwars` are both fine)
it's clear that if there are `**` then it's keyword arguments

File: schema.graphql [hunk 3]
```diff
@@ -31,6 +32,135 @@ type ImageType {
...
+  """
+  Movie's runtime in seconds
+  """
+  runtime: Int
+
+  """
+  Movie's title
+  """
+  title: String!
+>>>>>>> Implement moviesSearch query
```
Comment (bl0rch1d) [inline schema.graphql]: oops

File: schema.graphql [hunk 4]
```diff
@@ -56,15 +186,24 @@ type Mutation {
 }
 
 """
+<<<<<<< HEAD
```
Comment (bl0rch1d) [inline schema.graphql]: oops [1]

File: schema.graphql [hunk 5]
```diff
@@ -56,15 +186,24 @@ type Mutation {
...
 An object with an ID to support global identification.
 """
 interface Node {
   """
   Globally unique identifier.
+=======
```
Comment (bl0rch1d) [inline schema.graphql]: oops [2]

File: schema.graphql [hunk 6]
```diff
@@ -56,15 +186,24 @@ type Mutation {
...
+An object with an ID to support global identification
+"""
+interface NodeInterface {
+  """
+  Globally unique identifier
+>>>>>>> Implement moviesSearch query
```
Comment (bl0rch1d) [inline schema.graphql]: oops [3]

File: schema.graphql [hunk 7]
```diff
@@ -56,15 +186,24 @@ type Mutation {
...
   """
   id: ID!
 }
 
+<<<<<<< HEAD
```
Comment (bl0rch1d) [inline schema.graphql]: oops [4]

File: db/schema.rb [hunk 8]
```diff
@@ -36,6 +36,23 @@
...
+
+  create_table "movies", force: :cascade do |t|
+    t.string "title", null: false
+    t.string "original_title"
+    t.string "overview"
+    t.integer "revenue"
+    t.integer "budget"
+    t.integer "runtime"
+    t.string "original_language"
+  end
```
Comment (bl0rch1d) [inline db/schema.rb]: add timestamps, please

File: app/graphql/types/base/field.rb [hunk 9]
```diff
@@ -4,10 +4,6 @@ module Types
   module Base
     class Field < GraphQL::Schema::Field
       argument_class Types::Base::Argument
-
-      def resolve_field(obj, args, ctx)
-        resolve(obj, args, ctx)
-      end
```
Comment (bl0rch1d) [inline app/graphql/types/base/field.rb]: Why you remove this?

File: app/graphql/resolvers/auth_base.rb (hunk 2 above)
Comment (bl0rch1d) [reply app/graphql/resolvers/auth_base.rb]: ok

File: app/graphql/types/base/field.rb (hunk 9 above)
Comment (yaorlov) [reply app/graphql/types/base/field.rb]: this was added by the graphql generator, but it causes an error
there is an issue for this in graphql ruby

**Pull request merged**